### PR TITLE
Fix remaining tests

### PR DESCRIPTION
**Pull request by Kotarski**

### CODE REVIEW

File: gator/specs/common.py
Comment (Kotarski) [inline gator/specs/common.py:1]: Dataclassified spec

File: gator/specs/jobs.py
Comment (Kotarski) [inline gator/specs/jobs.py:1]: Dataclassified spec

File: gator/specs/resource.py
Comment (Kotarski) [inline gator/specs/resource.py:1]: Dataclassified spec

File: gator/wrapper.py
Comment (Kotarski) [inline gator/wrapper.py:1]: Env variables weren't expanding within command itself

File: mkdocs.yml
Comment (Kotarski) [inline mkdocs.yml:1]: As directed by doc builds warning

File: tests/common/test_db.py
Comment (Kotarski) [inline tests/common/test_db.py:1]: Pytest complains that class is marked with async, but method is not async

File: tests/test_local_scheduler.py
Comment (Kotarski) [inline tests/test_local_scheduler.py:1]: Fixup tests by passing through logger

File: tests/test_tier.py
Comment (Kotarski) [inline tests/test_tier.py:1]: Job doesn't fail on bad exit code - is this expected?

Comment (Intuity) [reply tests/test_tier.py:1]: That's probably not good

**Review by Intuity — APPROVED** [review]

File: tests/test_tier.py
Comment (Kotarski) [reply tests/test_tier.py:1]: Yea, I was thinking expected error code could be encoded in the spec potentially as (default 0)

**Pull request merged**